tests-functional/tests/test_wakuext_message_reactions.py

```diff
@@ -106,3 +107,71 @@ def test_one_to_one_message_reactions(self, backend_new_profile, waku_light_clie
                     item["emoji"] == "🙂",
                 )
             )
+
+    @pytest.mark.parametrize("waku_light_client", [False], indirect=True, ids=["wakuV2LightClient_False"])
```

Comment:
Does it not work with light client? 🤔

Reply:
I just felt that it was useless to test both light and full clients. I guess we can remove the parametrize, but I didn't know how to do that :sweat_smile: